app/about/layout.tsx

```diff
@@ -0,0 +1,71 @@
+import getQueryClient from "@/components/customs/getQueryClient";
```

Comment:
This is mostly the same as base-layout, made a few changes (like getting rid of the arbo navigation component since it was an unnecessary abstraction. I also added the state handling into the Nav component itself so we do not need to pass any props through.

Reply:
Not yet sure if that is the 100% best way to do this, but it is ok for now I think so